### PR TITLE
Logout in navbar

### LINKED ISSUE #128: Logout button in navbar when logged in
# Expected Behavior

Logout should be displayed in navbar when user is logged in.

# Current Behavior

Currently when user is logged in, navbar stays static and shows login and sign up

### PR DESCRIPTION
**Pull request by sweezy793**
Logout option is now available in navbar when a user is logged in

screenshot_20190307_221510

**Closing issues**

Closes #128

### CODE REVIEW

**Comment by sweezy793** [comment]
@ivantha Please review the changes :)

**Comment by sweezy793** [comment]
@root5533 Please review the changes.

**Comment by sweezy793** [comment]
@root5533 please review the changes.

File: fact-bounty-client/src/shared/components/TopNavBar.js [hunk 1]
```diff
@@ -1,75 +1,117 @@
...
+  }
+};
 
 class TopNavBar extends Component {
+  onLogoutClick = e => {
+    e.preventDefault();
+    this.props.logoutUser();
+  };
+
+  state = {
```
Comment (root5533) [inline fact-bounty-client/src/shared/components/TopNavBar.js]: Create state inside `constructor`

File: fact-bounty-client/src/shared/components/TopNavBar.js [hunk 2]
```diff
@@ -1,75 +1,117 @@
...
+  };
 
-	handleClose = () => {
-		this.setState(state => ({
-			anchorEl: null
-		}))
-	}
+  render() {
+    const { isAuthenticated} = this.props.auth;
+    const { anchorEl } = this.state;
```
Comment (root5533) [inline fact-bounty-client/src/shared/components/TopNavBar.js]: This variable is not used. Why is it here ?

File: fact-bounty-client/src/shared/components/TopNavBar.js (hunk 1 above)
Comment (sweezy793) [reply fact-bounty-client/src/shared/components/TopNavBar.js]: Will do :)

File: fact-bounty-client/src/shared/components/TopNavBar.js (hunk 2 above)
Comment (sweezy793) [reply fact-bounty-client/src/shared/components/TopNavBar.js]: > This variable is not used. Why is it here ?

{anchorEl} was already present in code base, but I will remove it now

**Pull request closed without merging**